### ISSUE TITLE
SHA-1 signing of webmin package not valid for RHEL9

### ISSUE THREAD

**Issue by L6TGKZ5A** [webmin/webmin#2010]
Not sure if this is a bug....

Just attempted to upgrade RHEL8 to RHEL9 and was advised that the current install of Webmin & Usermin would prevent the upgrade due to the packages being signed with an SHA-1 signature.  SHA-1 is deprecated in RHEL9.
Are new Webmin & Usermin packages, with updated signatures going to be available?

**Comment by iliajie** [comment]
> Are new Webmin & Usermin packages, with updated signatures going to be available?

Yes, and you can already update repos using `setup-repos.sh` script.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
